### PR TITLE
Spy Parrot

### PR DESCRIPTION
**Pull request by borisrev**
Self exparrotatory

### CODE REVIEW

**Comment by jmhobbs** [comment]
I love this.

File: parrots.json [hunk 1]
```diff
@@ -468,6 +468,10 @@
       "gif": "brazilianfanparrot.gif",
       "name": "Brazilian Fan Parrot"
     },
+    {
+      "gif": "spyparrot.gif",
+      "name": "Spy Parrot"
```
Comment (hugoheijboer) [inline parrots.json]: aussieparrot
spyparrot

File: parrots.json [hunk 2]
```diff
@@ -468,6 +468,10 @@
       "gif": "brazilianfanparrot.gif",
       "name": "Brazilian Fan Parrot"
     },
+    {
+      "gif": "spyparrot.gif",
```
Comment (hugoheijboer) [inline parrots.json]: spyparrot

File: parrots.json [hunk 3]
```diff
@@ -468,6 +468,10 @@
       "gif": "brazilianfanparrot.gif",
```
Comment (hugoheijboer) [inline parrots.json]: brazilianfanparrot

**Pull request merged**